roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -0,0 +1,61 @@
+# This file is included both in the openshift_master role and in the upgrade
+# playbooks.  For that reason the ha_svc variables are use set_fact instead of
+# the vars directory on the role.
+
+- debug: var=openshift_version
+
+- name: Init HA Service Info
+  set_fact:
+    ha_svc_svc_dir: "/usr/lib/systemd/system"
```

Comment:
Well, as part of that work I was wanting to find a clever way to make 'systemctl start atomic-openshift-master' do the right thing so that starting services is common whether HA or not. Actually the shortest path to this may just be to always run the split services no matter whether we're HA or not.